app/helpers/repository_helper.rb

```diff
@@ -0,0 +1,28 @@
+# Returns information about the repository that is configured in the
+# configuration file
+module RepositoryHelper
+  # Returns configuration properties for the current configured
+  # repository
+  def repository_config
+    conf = {}
+    conf['IS_REPOSITORY_ADMIN'] =
+        MarkusConfigurator.markus_config_repository_admin?
+    conf['REPOSITORY_PERMISSION_FILE'] =
+        MarkusConfigurator.markus_config_repository_permission_file
+    conf['REPOSITORY_STORAGE'] =
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.